objects.go

```diff
@@ -198,3 +198,37 @@ func (v *EADefListValue) UnmarshalJSON(b []byte) (err error) {
 	*v = EADefListValue(m["value"])
 	return
 }
+
+type RequestBody struct{
+	Data map[string]interface{} `json:"data,omitempty"`
+	Args map[string]string `json:"args,omitempty"`
+	Method string `json:"method"`
+	Object string `json:"object,omitempty"`
+	EnableSubstitution bool `json:"enable_substitution,omitempty"`
+	AssignState map[string]string `json:"assign_state,omitempty"`
+	Discard bool `json:"discard,omitempty"`
+}
```

Comment:
No there are no such restrictions on the fields. https://10.120.21.150/wapidoc/objects/request.html